packages/snaps-sdk/src/index.ts

```diff
@@ -1 +1,2 @@
+export * from './internals';
```

Comment:
Could we be more selective in our export? I don't expect that we need to export all internals?